templates/print-order/simple/receipt/print-content.php

```diff
@@ -205,6 +205,13 @@
 					if ( ! $product ) {
 						continue;
 					}
+					$original_qty = $item->get_quantity();
+					$qty_refunded = $order->get_qty_refunded_for_item( $item_id );
+					$adjusted_qty = $original_qty + $qty_refunded;
+					if ( $adjusted_qty <= 0 ) {
+						continue;
+					}
+					$item['qty'] = $adjusted_qty;
```

Comment:
Same code is used in all templates. Can we create a common function to get the adjusted qty and use in all places?